gcc/rust/rust-session-manager.cc

```diff
@@ -399,6 +399,12 @@ Session::enable_dump (std::string arg)
 void
 Session::handle_input_files (int num_files, const char **files)
 {
+  if (num_files == 0)
+    {
+      static const char *stdin_file[] = {"-"};
```

Comment:
does this work? Doesnt this just cause an fopen on "-"? Shouldnt there be a catch in our lexer for "-" to become stdin or something?